src/react-apps/applications/shared/__tests__/profileMenu.test.tsx

```diff
@@ -0,0 +1,76 @@
+import { shallow } from 'enzyme';
+import * as React from 'react';
+import ProfileMenuComponent from '../src/navigation/main-header/profileMenu';
+
+describe('ProfileMenu', () => {
+  let profileMenu: any;
+  let rendered: any;
+
+  beforeEach(() => {
+    profileMenu = null;
+    rendered = null;
```

Comment:
Since you only set the value to null, i think that you can declare the values in each test.

Because this will be used as expected values vs actuall value. Not props sent down to the component on render.

Reply:
renamed to afterEach() for cleanup after tests.